pages/devel-version.html

```diff
@@ -156,7 +156,7 @@ <h5>Quickstart Installation Instructions</h5>
                 <p>Please note Uyuni requires FQDNs for the Server, Proxy and clients, so usage of a properly-configured DNS server is mandatory for client updates to work.</p>
                 <p>If you want to install Uyuni in lab mode, for testing, you may be interested in how to easily make use of libvirt's included dnsmasq DNS and DHCP server: <a href="https://github.com/uyuni-project/uyuni/wiki/Libvirt-DNS-and-DHCP-without-Avahi">Libvirt DNS and DCHP without Avahi</a>.</p>
 
-                <p>1. Update openSUSE Leap 15.4:</p>
+                <p>1. Update openSUSE Leap 15.5:</p>
```

Comment:
I think you missed changing the memory requirement in this page.

Reply:
Changed